eng/Subsets.props

```diff
@@ -35,10 +35,13 @@
       You can get around this limitation by running the corehost subset, manually copying host
       artifacts to the test layout, then running the test subset.
   -->
+  <PropertyGroup>
+    <TargetsMobile Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'Android' or '$(TargetOS)' == 'tvOS'">true</TargetsMobile>
```

Comment:
This should be moved to https://github.com/dotnet/runtime/blob/master/eng/Configurations.props

Reply:
Thanks!